### DISCUSSION TITLE
SAE geeignet?

### DISCUSSION THREAD

**Discussion by Wildfire0815** [camueller/SmartApplianceEnabler#471] [Allgemein]
Hallo,
leider bin ich erst vor wenigen Tagen über den SAE gestolpert.
Auch wenn ich jetzt schon 2 Tage hier gelesen habe, kann ich mir die Frage ob der SAE für mich geeignet ist, noch nicht beantworten.

Mein Set-Up: 
•	Solaranlage mit STP 10.0 AV und SHM2.0 (Installateurs-PW vorhanden)
•	Wärmepumpe mit Modbus-Schnittstelle (nicht SG-ready, einzelne Parameter müssen als selbstständig angepasst werden)
•	Kaskadierte Zählerschaltung für Wärmepumpe, also vom Netz gesehen: Hauptzähler, SHM, Abzweig zur WP, Unterzähler für das Haus, Haus
•	IR-Zähler am Unterzähler für das Haus
•	Iobroker auf Synology mit funktionierenden Adaptern zu SHM/Wechselrichter, Wärmepumpe und IR-Lesekopf. Leistung der Wärmepumpe wird im Iobroker berechnet.
 
Geplante Erweiterung (in der Beschaffung/Umsetzung):
•	zweiter SMA-Wechselrichter(Hybrid) mit 16.6 kW Speicher
•	größere Balkonsolaranlage mit Shelly 1 plus PM als „Einspeisezähler“
•	Laden eines Plug-in Hybrid-PKW (Auto bestellt, bisher nur 16A-Drehstrom Steckdose ohne Zähler und Schalter vorhanden)
 
Ziel:
•	Möglichst wenig Strom zukaufen
 
Ist der SAE für diese Konstellation geeignet?
Kann ich den SAE vorerst parallel zum Iobroker auf der Synology installieren, bis der RPI wieder verfügbar ist?
Wie nähere ich mich am strukturiertesten dem SAE?
 
Viele Grüße,
 
Martin

**Comment by camueller** [comment]
> Möglichst wenig Strom zukaufen

Das Steuern macht der Sunny Home Manager. Der SAE erlaubt das Messen und Steuern von Geräten, die der Sunny Home Manager nicht direkt untersützt.

> Kann ich den SAE vorerst parallel zum Iobroker auf der Synology installieren, bis der RPI wieder verfügbar ist?

Ja, auch dauerhaft, wenn keine GPIOs benötigt werden.

> Wie nähere ich mich am strukturiertesten dem SAE?

Dokumentation lesen.
